.github/workflows/contrib-tests.yml

```diff
@@ -8,7 +8,9 @@ on:
     branches: ['main', 'dev/v0.2']
     paths:
       - 'autogen/**'
+      - 'autogen/img_utils.py'
       - 'test/agentchat/contrib/**'
+      - 'test/test_img_utils.py'
```

Comment:
Isn't this test run by another workflow?

Reply:
@sonichi As discussed, moved the img_utils into the contrib folder now.